src/sentry/sentry_apps/models/sentry_app.py

```diff
@@ -34,25 +35,72 @@
 from sentry.types.region import find_all_region_names, find_regions_for_sentry_app
 from sentry.utils import metrics
 
+
+class SentryAppResourceType(StrEnum):
+    ISSUE = "issue"
+    ERROR = "error"
+    COMMENT = "comment"
+    INSTALLATION = "installation"
+    METRIC_ALERT = "metric_alert"
+
+    # Represents an issue alert resource
+    EVENT_ALERT = "event_alert"
+
+
+class SentryAppActionType(StrEnum):
+    # Generic actions
+    CREATED = "created"
+    DELETED = "deleted"
+    UPDATED = "updated"
+
+    # Issue actions
+    ASSIGNED = "assigned"
+    IGNORED = "ignored"
+    RESOLVED = "resolved"
+    UNRESOLVED = "unresolved"
+
+    # Issue alert actions
+    TRIGGERED = "triggered"
+
+    # Metric alert actions
+    OPEN = "open"
+    CRITICAL = "critical"
+    WARNING = "warning"
```

Comment:
Rather than bundling all of these enums together, would it make more sense to have a separate enum, per domain? Like `IssueActionType`, `MetricAlertActionType` etc.